packages/wonder-stuff-ci/src/get-mobile-release-logger.ts

```diff
@@ -7,50 +7,36 @@ import {
 import * as winston from "winston";
 
 import {getGCPLogTransport} from "./get-gcp-log-transport";
-import {GCPLogLevels} from "./types";
+import {MobileReleaseLoggerOptions} from "./types";
 
 let logger: winston.Logger | null = null;
 
 /**
- * Logger for auditing mobile release events.
- * @param {boolean} redirectToStdout If true, logs will be written to stdout.
- * @param {{[key: string]: string}} labels K/V metadata that will be add to the logs
- * @param {GCPLogLevels} logLevel The log level to send to GCP.
- * @param {string} logName The name of the log to send logs to in GCP.
- * @param {string} projectId The GCP project ID to send logs to.
+ * Logger for auditing mobile release events with default values.
  */
-export const getMobileReleaseLogger = (
-    {
-        redirectToStdout,
-        labels,
-        logLevel,
-        logName,
-        projectId,
-    }: {
-        redirectToStdout: boolean;
-        labels: {[key: string]: string};
-        logLevel: GCPLogLevels;
-        logName: string;
-        projectId: string;
-    } = {
-        redirectToStdout: false,
-        labels: {},
-        logLevel: "info",
-        logName: "release-raccoon",
```

Comment:
This looked very gnarly for a function signature, so I turned it into a type to make it more readable.